cuda/test/matrix/dense_kernels.cpp

```diff
@@ -98,10 +106,10 @@ class Dense : public ::testing::Test {
 
     void set_up_apply_data()
     {
-        x = gen_mtx<Mtx>(40, 25);
-        c_x = gen_mtx<ComplexMtx>(40, 25);
+        x = gen_mtx<Mtx>(65, 25);
+        c_x = gen_mtx<ComplexMtx>(65, 25);
         y = gen_mtx<Mtx>(25, 35);
-        expected = gen_mtx<Mtx>(40, 35);
+        expected = gen_mtx<Mtx>(65, 35);
```

Comment:
Out of curiosity, why did you change these numbers? I am fine with the change, I would like to know why you did it.

Reply:
I did it because the default slice size is 64 and I wanted to test with at least two slices.

Reply:
Makes sense.